### ISSUE TITLE
why is request still blocked by CORS even though I have set alloworigin configuration?

### ISSUE THREAD

**Issue by syahnetwork** [prest/prest#374]
- pREST version (or commit ref): v0.3.4
OS/ PG : psql (PostgreSQL) 12.2 (Ubuntu 12.2-2

As the title says, I'm running svelte app and try to get data with below address. I already put them to allow origin on cors configuration in the .toml file

please help or direction, below is the screenshot , thank you :+1: 

![Screenshot from 2020-07-28 12-29-57](https://user-images.githubusercontent.com/45051925/88724158-40a16b00-d154-11ea-8d1d-42b01aa92013.png)

**Comment by syahnetwork** [comment]
@tiaguinho
@avelino 

My apologies for tagging :)

**Comment by tiaguinho** [comment]
@syahnetwork have you tried to do the request with postman? Did you receive the same error?

**Comment by syahnetwork** [comment]
@tiaguinho yes I have succeeded add data using postman by sending JSON, without error sir

**Comment by tiaguinho** [comment]
This may be happening because your svelte app doesn't send the header ```Access-Control-Allow-Origin```.
You can test this removing the ***cors*** configuration from your ***prest.toml***.

If it starts to work, probably the problem is on your svelte app.

Please, let us know what happens after this little test.

**Comment by avelino** [comment]
@syahnetwork more about CROS: https://postgres.rest/cors-support/
see if it works, in case it doesn't work I'm willing to fix the bug

**Comment by avelino** [comment]
@syahnetwork worked?
